app/helpers.py

```diff
@@ -1,22 +1,34 @@
 import yaml
 import datetime
+from app import models
 
+# For use in templates.
 def register(app):
 
   ###
   # Context processors and filters.
 
+  def scan_date():
+    return models.Report.report_time(models.Report.latest()['report_date'])
+
   # Make site metadata available everywhere.
   meta = yaml.safe_load(open("meta.yml"))
   @app.context_processor
   def inject_meta():
-      return dict(site=meta, now=datetime.datetime.utcnow)
+      return dict(site=meta, now=datetime.datetime.utcnow, scan_date=scan_date())
```

Comment:
This means that every web request triggers at least one database query, to get the latest scan date.